packages/tex-to-myst/src/marks.ts

```diff
@@ -53,6 +54,11 @@ const TEXT_MARKS_HANDLERS: Record<string, Handler> = {
     state.data.openGroups.push('inlineCode');
     state.data.ignoreNextWhitespace = true;
   },
+  macro_textrm(node, state) {
+    state.openParagraph();
```

Comment:
This only opens a paragraph if it's not already inside phrasing content 👍

However... looks like in tex this is often used inside math. Will that work here? I guess that will hopefully just need to be handled later in the math rendering.

Reply:
In a math context we take a different path and lift the original latex out, and don't process it (leaving that to katex/mathjax etc.).